opal/mca/patcher/base/patcher_base_patch.c

```diff
@@ -107,7 +107,10 @@ static void flush_and_invalidate_cache (unsigned long a)
 #elif OPAL_ASSEMBLY_ARCH == OPAL_IA64
     __asm__ volatile ("fc %0;; sync.i;; srlz.i;;" : : "r"(a) : "memory");
 #elif OPAL_ASSEMBLY_ARCH == OPAL_ARM64
-    __asm__ volatile ("dsb sy");
+    __asm__ volatile ("dc cvau, %0\n\t"
+                      "dsb ish\n\t"
+                      "ic ivau, %0\n\t"
+                      "dsb ish\n\t" :: "r" (a));
```

Comment:
@hjelmn few comments. 
```dsb ish``` -> ```dsb ish; isb```
it is not that obvious but ```isb``` (instruction barrier) is required here because DSB does not impose order on all instruction but memory access only. We don't want some thing like jump executed out of order so we have to use ISB in addition to DSB.

Reply:
Ok, will add isb.